### ISSUE TITLE
Create rule S1077 (`jsx-a11y/alt-text`): Image, area, button with image and object elements should have an alternative text

### ISSUE THREAD

**Issue by ilia-kebets-sonarsource** [SonarSource/SonarJS#4378]
use S1077 for JS, create ticket for `sonar-html` to extend scope to same as JS implementation

[plugin rule](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/alt-text.md#jsx-a11yalt-text)

[RSPEC PR](https://github.com/SonarSource/rspec/pull/3442)

[implementation PR](https://github.com/SonarSource/SonarJS/pull/4391)

[sonar-HTML issue](https://sonarsource.atlassian.net/browse/SONARHTML-181)
